pipelines/run-generic-pipelines-on-kubeflow-pipelines/README.md

```diff
@@ -0,0 +1,286 @@
+
+   If desired, you can visualize results directly in the Kubeflow Pipelines UI. For example, if a notebook trains a classification model, you could visualize its accuracy using a confusion matrix by producing metadata in Kubeflow Pipelines output viewer compatible format.
+
+1. Wait until node processing has completed before continuing.
+
+1. Open the _Visualizations_ tab. For illustrative purposes the first node that runs the `load_data` notebook produces metadata in markdown format, which identifies the data set download location.
+
+   ![Review visualizations](doc/images/review-visualization-in-kfp-gui.png)
+
+   > The code that produces the metadata is located in the notebook's last code cell.
+
+   Refer to [_Visualizing output from your notebooks or Python scripts in the Kubeflow Pipelines UI_](https://elyra.readthedocs.io/en/stable/recipes/visualizing-output-in-the-kfp-ui.html) to learn more about adding visualizations.
+
+1. Wait for the pipeline run to finish.
+
+   ![Wait for pipeline run to finish](doc/images/pipeline-run-completed.png)
+
+### Access the pipeline run outputs
+
+Pipelines that execute on Kubeflow Pipelines store the pipeline run outputs (completed notebooks, script output, and declared output files) in the cloud storage bucket you've configured in the runtime configuration.
+
+1. Open the object storage link and, if required, log in. 
+
+1. Navigate to the bucket that you've specified in the runtime configuration to review the content. Note that the bucket contains a "directory" with the pipeline's Kubeflow run name, e.g. `hello-generic-world-0716111722`.
+
+   ![Review run output artifacts](doc/images/inspect-object-storage-bucket-content.png)
+
+   If pipeline execution completed successfully the bucket contains for each node the following artifacts:
+      - a `tar.gz` archive containing the original notebook or script and, if applicable, its declared file dependencies
+      - if the node is associated with a notebook, the artifacts include the completed notebook with it's populated output cells and an HTML version of the completed notebook
+      - if the node is associated with a script, the artifacts include the console output that the script produced
+      - if applicable, the declared output files 
+
+   For example, for the `load_data` notebook the following artifacts should be present:
+
+   - `load_data-<UUID>.tar.gz` (input artifacts)
+   - `load_data.ipynb` (output artifact) 
+   - `load_data.html` (output artifact) 
+   - `data/noaa-weather-data-jfk-airport/jfk_weather.csv` (output artifact)
+
+1. Download the output artifacts to your local machine and inspect them.
+
+## Export the pipeline to a Kubeflow Pipelines format
+
+When you run a pipeline from the pipeline, Elyra compiles the pipeline, uploads the compiled pipeline, creates an experiment, and runs the experiment. If you want to run the pipeline at a later time outside of Elyra, you can export it.
+
```

Comment:
I think this is supposed to say `When you run a pipeline from the pipeline editor,`

Reply:
Jap. Fixed!